maptools: build: fix parallel build on older CMake

arnout opened the pull request:
The -j option for cmake --build was only introduce in CMake 3.12. To
support older CMake versions, replace it with a -j option passed after
the --.

Unfortunately, ninja doesn't support -j without argument. Therefore,
explicitly specify the number of jobs. One more than the number of CPUs
sounds like a good starting point.

Note that this limits the build engines to Ninja and Make, because
others don't know the -j option at all.

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>

tomereli commented on commands/build.py:
```diff
@@ -57,8 +58,11 @@ def prepare(self):
-               "--target", "install",
-               "-j" if not self.make_verbose else "-v"]
+               "--target", "install"]
+        if self.make_verbose:
```
make_verbose flag is broken (due to the --target change). The following works with ninja:
```
if self.make_verbose:
    cmd.extend(["--", "-v"])    
else:
    cmd.extend(["--", "-j", str(multiprocessing.cpu_count() + 1)])
```

But fails for make, no -v there - need to use VERBOSE=1 instead.

arnout replied:
Grmbl... `-v` was introduced in CMake 3.14, apparently...

I'll just convert it to VERBOSE=1 which is only supported in Make. If you need verbose build, you need to use make. Sounds fair.

arnout committed:
maptools: build: fix parallel build on older CMake

The -j option for cmake --build was only introduced in CMake 3.12. To
support older CMake versions, replace it with a -j option passed after
the --.

Unfortunately, ninja doesn't support -j without argument. Therefore,
explicitly specify the number of jobs. One more than the number of CPUs
sounds like a good starting point.

Note that this limits the build engines to Ninja and Make, because
others don't know the -j option at all.

Similarly, the -v option for cmake --build was only introduced in CMake
3.14. There is no easy solution for that one, so keep using the original
VERBOSE=1 there. This breaks the ninja build, so verbose has to be done
with make.

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
